presto-orc/src/main/java/com/facebook/presto/orc/block/BooleanBlockReader.java

```diff
@@ -30,8 +30,8 @@
 import static com.google.common.base.MoreObjects.toStringHelper;
 import static com.google.common.base.Preconditions.checkNotNull;
 
-public class BooleanJsonReader
```

Comment:
@dain can you look at the changes to the ORC code? I'm not very familiar with it